org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/rename/LSPRenameProcessor.java

```diff
@@ -197,7 +197,7 @@ public Change createChange(IProgressMonitor pm) throws CoreException, OperationC
 			throw new CoreException(
 					new Status(IStatus.ERROR, LanguageServerPlugin.PLUGIN_ID, Messages.rename_processor_required));
 		}
-		return LSPEclipseUtils.toCompositeChange(rename);
+		return LSPEclipseUtils.toCompositeChange(rename, "Rename"); //$NON-NLS-1$

```

Comment:
Shouldn't it be internationalized?

Reply:
Definitely should be internationalized. Done.